.pipelines/.vsts-vhd-builder-release-windows.yaml

```diff
@@ -102,6 +110,7 @@ stages:
       build23H2gen2: ${{ parameters.build23H2gen2 }}
       build2025:  ${{ parameters.build2025 }}
       build2025gen2:  ${{ parameters.build2025gen2 }}
+      windowsContainerImageJsonUrl: ${{ parameters.windowsContainerImageJsonUrl }}
```

Comment:
Here you could use a condition with `useContainerUrlsFromJson`.  If `true`, set windowsContainerImageJsonUrl to the windowsContainerImageJsonUrl, otherwise set to empty string.

You can then remove all other refs to useContainerUrlsFromJson